src/forms/reset-password-popup/reset-password/tests/ResetPasswordSuccess.test.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+
+import { render } from '@testing-library/react';
+
+import ResetPasswordSuccess from '../components/ResetPasswordSuccess';
+import messages from '../messages';
+
+// Mock defineMessages
+jest.mock('@edx/frontend-platform/i18n', () => ({
+  useIntl: () => ({
+    formatMessage: jest.fn((message) => message.defaultMessage),
+  }),
+  // eslint-disable-next-line no-shadow
+  defineMessages: (messages) => messages,
+}));
+
+describe('ResetPasswordSuccess component', () => {
+  it('renders success alert with correct message', () => {
```

Comment:
Could you explain this unit test?

Reply:
It verifies that the ResetPasswordSuccess component renders a success alert with the correct message

Reply:
Where we are checking success alert exists.
It should be like `expect(getByText(successMessage)).toBeTruthy();`

Reply:
> Where we are checking success alert exists. It should be like `expect(getByText(successMessage)).toBeTruthy();`

Done